Python/Product/Django/Django.csproj

```diff
@@ -113,6 +113,31 @@
     </Reference>
     <Reference Include="WindowsBase" />
   </ItemGroup>
+  <Choose>

```

Comment:
This is long enough, and repeats often enough, that I wonder if perhaps it should have its own .proj file that everyone can then just import?